src/akkudoktoreosserver/flask_server.py

```diff
@@ -132,9 +140,13 @@ def flask_gesamtlast():
 @app.route("/gesamtlast_simple", methods=["GET"])
 def flask_gesamtlast_simple():
     if request.method == "GET":
-        year_energy = float(
-            request.args.get("year_energy")
+        year_energy = request.args.get(
```

Comment:
to me, the name year_energy is ambiguous... also comment is misleading. 
Also, what type of energy is it?: consumed, produced, is it the actual measured energy or is it predicted?

Question: what is the endpoint queried for?
If it is the predicted energy demand for a given year, then I propose the name predicted_annual_energy_demand  or if it is the year then "predicted_annual_energy_demand_for_year" would be precise (and frank speaking a bit bulky; but hey, bits are cheap these days).

Please beg with me, in case I'm completely off topic here... I haven't reviewed the entire code yet.

Reply:
to be honest: i don't know what this is used for. It was dead for months and nobody complained. i thought about removing it completely. Ill look into the variable names and find better.